cluster/addons/cluster-monitoring/google/heapster-controller.yaml

```diff
@@ -37,6 +37,9 @@ spec:
       labels:
         k8s-app: heapster
         version: v1.1.0
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ''
```

Comment:
+1 for having this set to true
more context https://github.com/kubernetes/kubernetes/pull/29195#discussion_r71993721

@davidopp @thockin